Podfile

```diff
@@ -3,13 +3,13 @@ inhibit_all_warnings!
 
 def common_pods
   pod 'Alamofire', '~> 4.4'
-  pod 'Anchorage', '~> 4.0'
+  pod 'Anchorage', :git => 'https://github.com/Raizlabs/Anchorage.git', :branch => 'feature/xcode-9'
   pod 'KeychainAccess', '~> 3.0'
   pod 'PromiseKit', '~> 4.0'
   pod 'Sourcery', '~> 0.6'
   pod 'SwiftLint', '~> 0.18'
   pod 'Swinject', '~> 2.0'
```

Comment:
Please update to the newest version [2.1.1](http://cocoadocs.org/docsets/Swinject/2.1.1/).